packages/framework/src/Controller/Admin/DefaultController.php

```diff
@@ -476,4 +482,83 @@ protected function checkAllSliderNumericValuesAreSet(): void
             );
         }
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    #[Route(path: '/dashboard/search/')]
+    public function searchAction(Request $request): Response
```

Comment:
_nitpick_ 
I would prefer a separate controller for this action. It seems independent of the `DefaultController`